aten/src/ATen/native/cuda/SummaryOps.cu

```diff
@@ -22,6 +22,7 @@ enum class CUDAHistogramMemoryType { SHARED, MULTI_BLOCK, GLOBAL };
 template <
```

Comment:
this file finally compiles. but there're still other compiling errors, and I suspect it's because of some missing configs, it seems like the `histc_cuda` for input type double/float did not get generated successfully? Here's the log,
```
aten/src/ATen/CUDADoubleType.cpp: In member function ‘virtual at::Tensor& at::CUDADoubleType::_th_histc_out(at::Tensor&, const at::Tensor&, int64_t, at::Scalar, at::Scalar) const’:
aten/src/ATen/CUDADoubleType.cpp:2065:93: error: ‘THCudaDoubleTensor_histc’ was not declared in this scope
     THCudaDoubleTensor_histc(globalContext().getTHCState(), result_, self_, bins, min_, max_);
                                                                                             ^
aten/src/ATen/CUDADoubleType.cpp: In member function ‘virtual at::Tensor at::CUDADoubleType::_th_histc(const at::Tensor&, int64_t, at::Scalar, at::Scalar) const’:
aten/src/ATen/CUDADoubleType.cpp:2076:93: error: ‘THCudaDoubleTensor_histc’ was not declared in this scope
     THCudaDoubleTensor_histc(globalContext().getTHCState(), result_, self_, bins, min_, max_);
```